lib/AST/CASTBridging.cpp

```diff
@@ -44,58 +44,116 @@ struct BridgedDiagnosticImpl {
 };
 } // namespace
 
+static inline BridgedDeclContext bridgedDeclContext(DeclContext *declContext) {
+  return {declContext};
+}
+
+static inline DeclContext *unbridged(BridgedDeclContext cDeclContext) {
+  return static_cast<DeclContext *>(cDeclContext.raw);
+}
```

Comment:
In Swift, these would make wonderful extensions on `BridgedDeclContext` and `DeclContext`. A shame we don’t have type extensions in C++. 😢